License headers:

arch/arm64/include/asm/unistd.h:
/* SPDX-License-Identifier: GPL-2.0-only */

include/uapi/asm-generic/unistd.h:
/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */

--- a/arch/arm/tools/syscall.tbl.orig	2021-09-22 21:11:30.978885817 -0700
+++ b/arch/arm/tools/syscall.tbl	2021-09-22 21:12:58.723888709 -0700
@@ -463,3 +463,4 @@
 447	common	futex_wait			sys_futex_wait
 448	common	futex_wake			sys_futex_wake
 449	common  futex_waitv                     sys_futex_waitv
+450	common	futex_requeue			sys_futex_requeue
--- a/arch/arm64/include/asm/unistd.h.orig	2021-09-22 21:11:30.979885817 -0700
+++ b/arch/arm64/include/asm/unistd.h	2021-09-22 21:13:23.265889517 -0700
@@ -38,7 +38,7 @@
 #define __ARM_NR_compat_set_tls		(__ARM_NR_COMPAT_BASE + 5)
 #define __ARM_NR_COMPAT_END		(__ARM_NR_COMPAT_BASE + 0x800)
 
-#define __NR_compat_syscalls		450
+#define __NR_compat_syscalls		451
 #endif
 
 #define __ARCH_WANT_SYS_CLONE
--- a/arch/x86/entry/syscalls/syscall_32.tbl.orig	2021-09-22 21:11:30.980885817 -0700
+++ b/arch/x86/entry/syscalls/syscall_32.tbl	2021-09-22 21:13:45.249890242 -0700
@@ -455,3 +455,4 @@
 448	i386	futex_wait		sys_futex_wait
 449	i386	futex_wake		sys_futex_wake
 450	i386	futex_waitv		sys_futex_waitv			compat_sys_futex_waitv
+451	i386	futex_requeue		sys_futex_requeue		compat_sys_futex_requeue
--- a/arch/x86/entry/syscalls/syscall_64.tbl.orig	2021-09-22 21:11:30.980885817 -0700
+++ b/arch/x86/entry/syscalls/syscall_64.tbl	2021-09-22 21:14:03.696890850 -0700
@@ -372,6 +372,7 @@
 448	common	futex_wait		sys_futex_wait
 449	common	futex_wake		sys_futex_wake
 450	common	futex_waitv		sys_futex_waitv
+451	common	futex_requeue		sys_futex_requeue
 
 #
 # Due to a historical design error, certain syscalls are numbered differently
--- a/include/uapi/asm-generic/unistd.h.orig	2021-09-22 21:11:30.981885817 -0700
+++ b/include/uapi/asm-generic/unistd.h	2021-09-22 21:14:30.398891730 -0700
@@ -887,8 +887,11 @@ __SYSCALL(__NR_futex_wake, sys_futex_wak
 #define __NR_futex_waitv 450
 __SC_COMP(__NR_futex_waitv, sys_futex_waitv, compat_sys_futex_waitv)
 
+#define __NR_futex_requeue 451
+__SC_COMP(__NR_futex_requeue, sys_futex_requeue, compat_sys_futex_requeue)
+
 #undef __NR_syscalls
-#define __NR_syscalls 451
+#define __NR_syscalls 452
 
 /*
  * 32 bit systems traditionally used different
--- a/tools/perf/arch/x86/entry/syscalls/syscall_64.tbl.orig	2021-09-22 21:11:30.982885817 -0700
+++ b/tools/perf/arch/x86/entry/syscalls/syscall_64.tbl	2021-09-22 21:14:50.957892407 -0700
@@ -372,6 +372,7 @@
 448	common  futex_wait              sys_futex_wait
 449	common  futex_wake              sys_futex_wake
 450	common  futex_waitv             sys_futex_waitv
+451	common  futex_requeue		sys_futex_requeue
 
 #
 # Due to a historical design error, certain syscalls are numbered differently
